src/Files.App/App.xaml

```diff
@@ -31,6 +31,10 @@
 				<Setter Target="HighContrastAdjustment" Value="None" />
 			</Style>
 
+			<Style TargetType="FontIcon">
```

Comment:
I don't think we want to target all font icons. It's best to do it as needed.

Reply:
But icons that use glyph are not rendered with correct foreground in context flyout with your idea.
If there’ll be a need to change color for specific items, we could set TextFillColorPrimaryBrush.
Even we don’t want to do this app-wide, could we add it to each layout page, sidebar, inner nav toolbar, address toolbar instead of setting to each?

Reply:
> But icons that use glyph are not rendered with correct foreground in context flyout with your idea.

Ideally we'd have a custom style for the menu items. This will also solve an issue with the font.

Reply:
All of them?
Then I should close this PR